temporal-testing/src/main/java/io/temporal/testing/WorkflowReplayer.java

```diff
@@ -172,6 +172,56 @@ public static void replayWorkflowExecution(WorkflowExecutionHistory history, Wor
     worker.replayWorkflowExecution(history);
   }
 
+  /**
+   * Replays workflows provided by an iterable.
+   *
+   * @param histories The histories to be replayed
+   * @param failFast If true, throws upon the first error encountered (if any) during replay. If
+   *     false, all histories will be replayed and the returned object contains information about
+   *     any failures.
+   * @return If `failFast` is false, contains any replay failures encountered.
+   * @throws Exception If replay failed and `failFast` is true.
+   */
+  public static ReplayResults replayWorkflowExecutions(
+      Iterable<WorkflowExecutionHistory> histories, boolean failFast, Class<?>... workflowClasses)
```

Comment:
Not sure if we want the fail fast flag in here, or more overloads, that way there can be ones that don't throw checked exceptions.